src/content/docs/infrastructure/host-integrations/host-integrations-list/dojo-io-integration.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Dojo.io integration
+tags:
+    - dojo.io integration
+    - dojo integration
+    - new relic integrations
+metaDescription: "The New Relic Dojo.io integration sends performance metrics and inventory data from your Dojo.io framework to the New Relic platform."
+---
+import dojodashboard from 'images/infrastructure_screenshot-crop_dojo-dashboard.png'
+
+With the help of New Relic's Dojo dashboard, you can track your application's core web vitals, user time on the site, and browser performance metrics like Initial page load, route change, throughput and page views with Javascript errors. Built with our infrastructure agent and our browser monitoring, the Dojo dashboard lets you view all your most important data in one place.
+
+<img
+  title="dashboard-example.png"
+  alt="A screenshot of the Dojo dashboard"
+  src={dojodashboard}
+/>
+
+## Install the infrastructure agent and Dojo integration [#install]
+
+1. Install the infrastructure agent (optional)
+Before getting Dojo data into New Relic, first install our infrastructure agent to get details about your host.
+* Follow our [guided install](https://one.newrelic.com/nr1-core?state=999174fa-ed8b-d707-af7b-8e4f8fd03cc1) to instrument your system with the infrastructure agent. Or if you prefer, you can install the infrastructure agent by [following the docs](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent).
+* By integrating this infrastructure agent in your system, you can see the information for the CPU usage, memory usage, storage usage, disk usage, network traffic, and more.
+2. Follow our [browser monitoring guide](/docs/browser/browser-monitoring/getting-started/browser-summary-page/) to instrument your Dojo application (or) follow this steps to [install the Browser Monitoring Agent](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent/).
+3. Monitor your metrics by using the dashboard template on one.newrelic.com.
```

Comment:
I think for this section, we should just use the boilerplate text we have for installing the dojo dashboards from IO. Something like:

> With the browser agent installed and instrumented with your app, you can view your raw data in Metrics & events. Our default dashboards transform that raw data into charts and graphs, which provide a bird’s eye view of your system’s health. To install our default dashboards, go to our [<APP NAME> instant observability page](LINK WILL BE PROVIDED AFTER DOC IS LIVE).

Reply:
Feel free to edit this text, but it's a little more fleshed out and give customers more direction than the original draft